services/QuillLMS/client/app/bundles/Connect/components/studentLessons/lesson.jsx

```diff
@@ -549,7 +556,7 @@ export class Lesson extends React.Component {
           translate={translate}
         />
       );
-    } else if (availableLanguages?.length > 1 && !language && !ALPHA_TRANSLATED_ACTIVITY_UIDS.includes(params.lessonID)) {
+    } else if (!playLesson?.language && sessionLanguageSet && availableLanguages?.length > 1 && !language && !ALPHA_TRANSLATED_ACTIVITY_UIDS.includes(params.lessonID)) {
```

Comment:
Can we just use `session` here rather than `sessionLanguageSet`, and not have to keep track of another variable?

Reply:
totally! that's a great suggestion